modules/tinymce/src/plugins/lists/main/ts/core/Util.ts

```diff
@@ -1,25 +1,47 @@
+import { Toolbar, Menu } from '@ephox/bridge';
```

Comment:
Bridge should never be used directly outside of `core` and `silver`. You should use the `Ui` module from TinyMCE core instead, for example:
```ts
import { Menu, Toolbar } from 'tinymce/core/api/ui/Ui';
```